include/proj/internal/coordinateoperation_constants.hpp

```diff
@@ -729,9 +741,15 @@ static const MethodMapping projectionMethodMappings[] = {
     {PROJ_WKT2_NAME_METHOD_STEREOGRAPHIC, 0, "Stereographic", "stere", nullptr,
      paramsObliqueStereo},
 
+    {PROJ_WKT2_NAME_METHOD_TIMES, 0, "Times", "times", nullptr,
+     paramsLonNatOrigin},
+
     {PROJ_WKT2_NAME_METHOD_VAN_DER_GRINTEN, 0, "VanDerGrinten", "vandg", "R_A",
      paramsLonNatOrigin},
 
+    {PROJ_WKT2_NAME_METHOD_VERTICAL_NEAR_SIDE_PERSPECTIVE, 0,
+     "Vertical_Near_Side_Perspective", "nsper", nullptr, paramsLonNatOrigin},
```

Comment:
paramsLonNatOrigin is not appropriate for nsper. nsper has a "h" mandatory parameter and lat_0 is also handled.